User_Story#1/SortingApp/SortinfApp.UnitTests/Utilities/DataReaderTest.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.IO;
+using BusinessLayer.Reader;
+using BusinessLayer.Utilities.Validator.Interfaces.Fakes;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace SortinfApp.UnitTests.Utilities
+{
+    /// <summary>
+    /// Represents tests for DataReader class.
+    /// </summary>
+    /// <owner>Anton Petrenko</owner>
+    [TestClass]
+    public class DataReaderTest
+    {
+        /// <summary>
+        /// Tests DataReader if pass empty validator in constructor it will throw ArgumentNullException.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        [TestMethod]
+        public void DataReader_DataReader_PassEmptyValidatorInConstructor_ThrowArgumentNullException()
+        {
+            //
+            // Arrange.
+            //
+            StubIValidator validator = null;
+
+            //
+            // Assert.
+            //
+            Assert.ThrowsException<ArgumentNullException>(() => new DataReader(validator));
+        }
+
+        /// <summary>
+        /// Tests ReadContent if pass empty path to the file it will throw ArgumentNullException.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        [TestMethod]
+        public void DataReader_ReadContent_PassEmptyPathToTheFile_ThrowArgumentNullException()
+        {
+            //
+            // Arrange.
+            //
+            var validator = new StubIValidator()
+            {
+                IsDataExistString = (string str) => true
```

Comment:
Just FYI, we can skip type declaration in lambda expression.
It should work in this way too:
IsDataExistString = (str) => true

But it is up to you to decide how to use that, perhaps some way is more clear for you.